gravitee-apim-portal-webui/src/app/components/gv-page-markdown/gv-page-markdown.component.css

```diff
@@ -100,3 +100,9 @@ app-gv-markdown-toc {
   color: var(--gv-theme-font-color-light, #ffffff);
   padding-top: 6px;
 }
+
+:host ::ng-deep table td {
```

Comment:
ha on :host. so we're sure that only this component uses this css. and not all the application's tables after opening this component? 

Usually i prefer to add explicite `app-gv-page-markdown` tag to be sure this css are on apply here 🤷‍♂️

Reply:
I checked and it's fine with me 🚀